docs/en/operation/Journal.md

```diff
@@ -216,6 +218,24 @@ run
 ./bin/alluxio-start.sh secondary_master
 ```
 
+#### Checkpointing on primary master
+
+Checkpointing requires a pause in master metadata changes and causes temporary service
+unavailability while the checkpoint is written. If Alluxio contains millions or billions of files, it may take hours
```

Comment:
It would be good to provide some range where people could have a quick judgement based on their data size.

Reply:
I tried 1 million file takes about 5s, but when trying 1 billion file... it failed for a strange exception 
```
org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.hdfs.server.namenode.LeaseExpiredException): No lease on /alluxio_journal/FileSystemMaster/v1/.tmp/50aa1066-5bec-434e-b481-10d9659b88de (inode 16478): File does not exist. [Lease.  Holder: DFSClient_NONMAPREDUCE_2121578221_57, pendingcreates: 1]
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkLease(FSNamesystem.java:3428)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.analyzeFileState(FSNamesystem.java:3233)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getNewBlockTargets(FSNamesystem.java:3071)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:3031)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:724)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:492)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:616)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:969)
	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:2049)
	at org.apache.hadoop.ipc.Server$Handler$1.run(Server.java:2045)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1657)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2043)
```
Will adding some time expectation

Reply:
@ggezer I preferred not to give numbers now since the number will not be correct and largely depends on the journal type and file metadata size. And also because... I only get the million file data which is not that useful.

Reply:
I see, makes sense. Then let's not mention file count in this directive as well. We could just say, "This operation may take hours depending on Alluxio namespace size."

Reply:
Changed! Thanks for your recommendation!